source/_integrations/rpi_camera.markdown

```diff
@@ -16,10 +16,16 @@ To enable this camera in your installation, add the following to your `configura
 
 ```yaml
 # Example configuration.yaml entry
-camera:
-  - platform: rpi_camera
+rpi_camera:
+    image_width: 1296
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
no problem, I removed optional parameters from the example